app/templates/views/pricing/index.html

```diff
@@ -28,29 +28,29 @@ <h1 class="heading-large">Pricing</h1>
 
   <p class="govuk-body">
     {% if not current_user.is_authenticated %}
-      <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.register') }}">Create an account</a> then add as many unique services as you need to.</p>
+      <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.register') }}">Create an account,</a> then add your unique service(s).</p>
     {% else %}
-      You can add as many unique services as you need to.
+      Additional unique services may be added, subject to approval by the U.S. Notify team.
```

Comment:
What do we imagine for approval? There's no current mechanism for this, so is it just individual manual follow-up?

Reply:
I think even after we more concretely outline criteria, it feels to me like it will always be manual?